zmq.descr

```diff
@@ -0,0 +1 @@
+OCaml bindings for ZeroMQ 4.0
```

Comment:
Maybe call it 4.x as it's not specific to 4.0?

Reply:
Changed it to 4.x